ui/qt-lib/src/Ariadne/UI/Qt/Widgets/WalletInfo.hs

```diff
@@ -268,8 +240,21 @@ requestButtonClicked langFace WalletInfo{..} _checked = do
       whenJustM (readIORef currentItem) $ \item -> do
         let
           accounts = case item of
-            WIWallet uacis -> RequestAccountsMulti uacis
-            WIAccount uaci -> RequestAccountsSingle uaci
+            WIWallet _ uacis -> RequestAccountsMulti uacis
+            WIAccount _ uaci -> RequestAccountsSingle uaci
         req' <- startRequest langFace onClosed accounts
         writeIORef requestDialog $ Just req'
   where onClosed = writeIORef requestDialog Nothing
+
+sendButtonClicked :: UiLangFace -> WalletInfo -> Bool -> IO ()
+sendButtonClicked UiLangFace{..} WalletInfo{..} _checked = whenJustM (readIORef currentItem) $ \item -> do
+  let
+    (wIdx, inputs) = case item of
+      WIWallet wIdx uacis -> (wIdx, SendInputsMulti uacis)
+      WIAccount wIdx UiAccountInfo{..} -> (wIdx, SendInputsSingle $ head uaciPath)
+  result <- runSend inputs
+  case result of
+    SendCancel -> return ()
+    SendSuccess SendOptions{..} -> langPutUiCommand (UiSend wIdx soAccounts soAddress soAmount) >>= \case
+      Left err -> void $ QMessageBox.critical walletInfo ("Error" :: String) $ toString err
```

Comment:
I'd display some kind of progress spinner here, need to discuss with @a-rukin. It takes some time between the dialog closes and backend does something with our transaction.

Reply:
Have you discussed it? Is it for later (some subsequent issue)?

Reply:
Updated AD-389.